### DISCUSSION TITLE
Unit Pricing

### DISCUSSION THREAD

**Discussion by SGFGOV** [medusajs/medusa#5126] [Feature Requests]
Is it possible for Medusa to provide unit prices ?
For example if my package contains content with net weight 10kgs and cost 12 Euros,
the per unit price will 1.2
Some markets and industries require unit pricing

**Comment by tsvetann** [comment]
What would the customer order in this case? Units?

**Reply by SGFGOV** [reply]
No the customer will continue to order standard packs. Say 5Kg and 10Kg, but by convention, the sales happens on a per unit price
Typically the unit price of a 10kg pack will be less than that of 5kg pack. 
A 10kg pack maybe 12 Euros whereas a 5 kg pack may be 8 Euros. I think german markets already mandate this unit pricing requirement and shopify implements it natively because of this reason

**Comment by bqst** [comment]
I agree, for now, I need to set up `weight` and `weightUnit` (ml, l, g, kg, etc.) as `metadata` fields for each Product or Variant and then implement a calculation method on the front end.
Note that I am not using the shipping `weight` variable of the Product or Variant because I think it is more suitable for calculating shipping fees based on weight with a fixed unit (such as kg).
